addon/eslint.config.mjs

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
Added to upgrade eslint 8 to 9 for the addon/ dir, copied with minor changes from test-app's eslint config